static/src/js/admin/insert_update.js

```diff
@@ -1285,6 +1337,56 @@ define(
                 initPreviewLayout();
             }
 
+            /**
+             * Implements the "change" event on the given wysiwyg
+             *
+             * @param $wysiwyg
+             * @return Promise
+             */
+            function observeWysiwygChange($wysiwyg) {
+                var deferred = jQuery.Deferred();
+
+                $wysiwyg.nosOnShow('one', function() {
+                    // Delays the execution of the following code, as the wysiwyg is initialized just after this script
+                    setTimeout(function() {
+                        // Waits for the wysiwyg plugin to be loaded
+                        require(['jquery-nos-wysiwyg'], function($) {
+                            $(function() {
+                                // Waits for the editor to be fully initialized
+                                setTimeout(function() {
```

Comment:
There seems to be events to properly detect when TinyMCE is loaded : https://www.tinymce.com/docs/advanced/events/#init

Reply:
I agree this would be more robust, bu in order to do that I would have to overload `Renderer_Wysiwyg` and rewrite the view `renderer/wysiwyg`. What you think ?